src/execute/oas3/parameter-builders.js

```diff
@@ -37,6 +37,9 @@ export function query({req, value, parameter}) {
     return
   }
 
+  // Encode query parameter name
+  parameter.name = encodeURIComponent(parameter.name)
```

Comment:
Instead of mutating the parameter's name here (which would probably cause some unintended consequences elsewhere in this library), I'd like to instead see `encodeURIComponent` being used to encode `parameter.name` where we're passing that value to `stylize()` further down in this function.

Reply:
I have moved the `encodeURIComponent` call closer to where the value is actually used.